packages/nextjs/views/ModalFiltersMobile.tsx

```diff
@@ -33,7 +36,7 @@ export const ModalFiltersMobile: React.FC<ModalFiltersMobileProps> = ({
             className="w-full rounded-2xl font-semibold justify-center"
             onClick={() => onClose(false)}
           >
-            View results
+            View results {total > 0 ? `(${total})` : ""}
```

Comment:
On the filters button, showing the number of active filters makes a lot of sense. Shown on this "View Results" button, however, I would assume any number that appears is telling me how many results matched the filters I selected. A mismatch in expectations there could be quite confusing.

Reply:
🤔 I think you are right, I got rid of the number